modules/administration-guide/pages/finding-the-fqdn-of-your-organizations-instance.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: PROCEDURE
+:description: Finding the FQDN of your organization's instance of {prod-short}
+:keywords: administration-guide, FQDN, URL
+:navtitle: Finding the FQDN of your organization's instance of {prod-short}
+//:page-aliases:
+
+[id="finding-the-fqdn-of-your-organizations-instance"]
+= Finding the FQDN of your organization's instance of {prod-short}
+
+You can get the FQDN of your organization's instance of {prod-short} on the command line or in the OpenShift web console.
+
+TIP: You can find the FQDN for your organization's {prod-short} instance in the *Administrator* view of the OpenShift web console as follows. Go to *Operators* -> *OperatorHub* -> *Installed Operators* -> *{prod} instance Specification* -> *{prod-checluster}* -> *{prod} URL*.
+
+Because the FQDN of {prod-short} is part of the dashboard URL, users get the FQDN whenever they open the dashboard.
```

Comment:
What do we mean by "users get the FQDN" here?

Reply:
I would remove the last sentence in general

Reply:
Removed.